docs/source/apps/cct.rst

```diff
@@ -48,8 +48,13 @@ file referenced by the {object_reference} must contain a valid
 {object_definition}.
 
     .. versionadded:: 8.0.0
-
-
+    
+Input should be lines of consisting of:
```

Comment:
unfortunately that is not all that simple. It could be longitude+latitude, or latitude+longitude, or easting+northing, or northing+easting. The units might be anything.  And for time it is not in seconds but in decimal year (like 2022.9)